dune-project

```diff
@@ -377,6 +377,27 @@ functions.
   )
 )
 
+(package
+  (name tracing)
+  (version "1.5.5")
+  (synopsis "Small library to help trace code")
```

Comment:
use (wrapped false)
and rename maybe basic.ml to Tracing.ml

Reply:
I assumed this comment wasn't meant to be on the top level dune-project?